graph/GuidelinesGraph.md

```diff
@@ -326,6 +327,78 @@ For a complete mapping of error codes to HTTP statuses, see
 
 <a name="api-contract-and-non-backward-compatible-changes"></a>
 
+### Enums
+
+In OData, enum are nominal types representing a subset of the nominal type they use, and are especially useful in cases where certain properties have predefined, limited options.
+
+```xml
+<EnumType Name="Color">
+    <Member Name="Red" Value="0" />
+    <Member Name="Green" Value="1" />
+    <Member Name="Blue" Value="2" />
+</EnumType>
+```
+
+#### Pros
+
+- Our SDK generators will translate the enum to the best representation of the target programming language, resulting in a better developer experience and free client side validation
+
+#### Cons
+
+- Adding a new value requires to go through a (generally fast) API Review
+- If the enum is not [evolvable](./patterns/evolvable-enums.md), adding a new value is a breaking change and will generally not be allowed
+
+#### Enum or Booleans
```

Comment:
Another thing we see a **lot** of, are that folks define collections of enum, rather than using flags enums.  Can we add something about that too please?

Also Garrett has previously pointed out that a nullable enum is an indicator that the enum should have an additional member for the null state.  Similarly, a nullable Boolean indicates tri-state, which should be modelled as an enum instead.